pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -150,14 +248,17 @@ def test_call_specific(self):
         # Reconfigure population and check that a person is able to progress
         # infcetion status mutliple times in the same time step. This will be
```

Comment:
```suggestion
        # infection status multiple times in the same time step. This will be
```
Knew I'd get you on the spelling somewhere ;)